model/colorspace.go

```diff
@@ -306,24 +306,27 @@ func (cs *PdfColorspaceDeviceGray) ColorToRGB(color PdfColor) (PdfColor, error)
 func (cs *PdfColorspaceDeviceGray) ImageToRGB(img Image) (Image, error) {
```

Comment:
What about other color spaces?  Are similar things needed there?

Reply:
I think it might be worth optimizing the `ImageToRGB` conversion of the `CMYK` colorspace. The other colorspaces don't seem too common and some of them are based on an underlying colorspace so they would most likely use the conversion method of one the `RGB`, `Gray` or `CMYK`  colorspaces.